### PR TITLE
Tips section for SSL certificate usage

### PR DESCRIPTION
**Pull request by mosabua**
https://issues.sonatype.org/browse/NEXUS-9664

### CODE REVIEW

**Comment by dulanism** [comment]
Sorry. I accidentally closed this while reading it.

**Comment by mosabua** [comment]
No worries @dulanism  .. want to review it? 

@rseddon @peterlynch you can also have a look. We can launch this onto the site on Monday and then if we feel we need more still add stuff.

File: chapter-docker.asciidoc [hunk 1]
```diff
@@ -60,6 +64,52 @@ instead of the default root context http://localhost:8081/, the URLs for your Do
 use the configured port for the repository and omit the context path in the URL. This is a side-effect of the the 
 fact that Docker does not support context paths in the registry API.
 
+==== Tips for SSL Certificate Usage
+
+The requirement of Docker to use HTTPS forces the usage of SSL certificates. By default, Docker looks up the
+validity of the certificate by checking with certificate authorities. If you are using a purchased certificate,
```
Comment (joedragons) [inline chapter-docker.asciidoc]: Minor: I know a great source of debate here but I don't think the comma after certificate is needed.  "purchased certificate that is registered" seems fine to me.

File: chapter-docker.asciidoc [hunk 2]
```diff
@@ -60,6 +64,52 @@ instead of the default root context http://localhost:8081/, the URLs for your Do
...
+that is registered with these authorities, all functionality works as desired. However, if you are creating a
+certificate yourself with tools such as `openssl`, it is not registered. This is often called a self-signed
+certificate. Using a self-signed certificate requires further configuration steps to ensure that Docker provides
+full functionality.
+
+TIP: SSL certificates are not expensive and we recommend buying a certicate for your {pro} to simplify Docker
+usage and increase security.
+
+Docker can be configured to skip certificate validation, which effectively allows usage of self-signed
+certificates, with the `--insecure-registry` flag. Each repository on the repository manager accessed from Docker,
```
Comment (joedragons) [inline chapter-docker.asciidoc]: in instead of on?

**Comment by joedragons** [comment]
+1 if this works.  I have on my todo to check it out but it's gonna be a while for me=\

**Comment by mosabua** [comment]
@joedragons  this is based on official docs and what @rseddon tested and found out as well as what I did during my writing with regards to testing. So there is no need for you to test it.

**Comment by joedragons** [comment]
To be clear, it was on my radar before this PR, so my test was not for this
PR but to see how it worked=)

**Comment by dulanism** [comment]
No contribution. Just actively reading to better understand how all this works @mosabua

File: chapter-docker.asciidoc (hunk 1 above)
Comment (mosabua) [inline chapter-docker.asciidoc]: Fixed.

File: chapter-docker.asciidoc (hunk 2 above)
Comment (mosabua) [inline chapter-docker.asciidoc]: Fixed.

**Pull request merged**